gallery/how_to/work_with_microtvm/micro_aot.py

```diff
@@ -30,16 +30,40 @@
 or on Zephyr platform on a microcontroller/board supported by Zephyr.
 """
 
+######################################################################
+#
+#     .. include:: ../../../../gallery/how_to/work_with_microtvm/install_dependencies.rst
+#
+
 # sphinx_gallery_start_ignore
 from tvm import testing
 
 testing.utils.install_request_hook(depth=3)
 # sphinx_gallery_end_ignore
 
+# You can skip the following two sections (installing Zephyr and CMSIS-NN) if the following flag is False.
+# Installing Zephyr takes ~20 min.
+import os
+
+use_physical_hw = bool(os.getenv("TVM_MICRO_USE_HW"))
```

Comment:
I think this could be worded / setup clearer for the user, since the code doesn't check `use_physical_hw` (I assume) it's unclear who is doing the skipping.

Reply:
Expanded on this comment a bit.